deployment.rb

```diff
@@ -46,6 +46,7 @@ def load_configuration
     'build_apps'                  => false,
     'build_dashboard'             => true,
     'build_pegasus'               => true,
+    'census_map_table_id'         => rack_env == :production ? '1AUZYRjLMI5NiQsDeDBGFsOIFpL_rLGsnxNpSyR13' : nil,
```

Comment:
Is there any downside to this being in the repo?

Reply:
I don't think so. You can see the value if you view source on the page.